### PR TITLE
ci: bump gui/headless to 1.1.10

### PR DESCRIPTION
**Pull request by jamilbk**
To publish the mpsc channel fix.

### CODE REVIEW

File: website/src/components/Changelog/Headless.tsx [hunk 1]
```diff
@@ -10,17 +10,23 @@ export default function Headless() {
   return (
     <Entries href={href} arches={arches} title="Linux headless">
       {/*
+      <Entry version="1.2.0" date={new Date("Invalid date")}>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+        TODO
+        </ul>
+      </Entry>
+      */}
       <Entry version="1.1.5" date={new Date("Invalid date")}>
```
Comment (ReactorScram) [inline website/src/components/Changelog/Headless.tsx]:
```suggestion
      <Entry version="1.1.5" date={new Date("2024-08-08")}>
```

**Review by ReactorScram — CHANGES_REQUESTED** [review]

**Comment by ReactorScram** [comment]
Oh yeah and why is the next version 1.2.x? Internet Resource?

File: website/src/components/Changelog/Headless.tsx (hunk 1 above)
Comment (jamilbk) [reply website/src/components/Changelog/Headless.tsx]: Good catch, thanks.

**Comment by jamilbk** [comment]
> Oh yeah and why is the next version 1.2.x? Internet Resource?

#6214

**Review by ReactorScram — APPROVED** [review]

**Pull request merged**